src/exif-reader.js

```diff
@@ -78,6 +79,15 @@ export function loadView(dataView, options = {expanded: false}) {
             tags = Object.assign({}, tags, readTags);
         }
     }
+    if (hasIccData(iccChunks)) {
+        foundMetaData = true;
+        const readTags = IccTags.read(dataView, iccChunks);
+        if (options.expanded) {
+            tags.icc = readTags;
+        } else {
+            tags = Object.assign({}, tags, readTags);
+        }
+    }
```

Comment:
Add this to the tests in `exif-reader-spec.js`.

Reply:
idk, loading full files is not really unit test any more but on the other hand it is fast enough to run in npm test watch mode and it increases confidence. I'd leave it here but that is mostly because I don't have a strong opinion :)